Tools/scripts/smelly.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-# Script checking if all symbols exported by libpython start with "Py" or "_Py"
+# Script checking that all symbols exported by libpython start with Py or _Py
```

Comment:
I would use `"Py" or "_Py"`  to keep the same style with others comments

Reply:
I tried, but it doesn't fit into 80 characters :-)

Reply:
You can write:
```
# Script checking that all symbols exported by libpython
# start with Py or _Py
```
Is just a style preference =D